demystifying_command_line/demystifying_command_line.md

```diff
@@ -0,0 +1,234 @@
+
+[[ ]] GUIs offer the same level of control of a system as command line interfaces (CLIs).
+[[X]] Pointing-and-clicking is a frequently used way of interacting with GUIs.
+[[X]] GUIs depend principally on visual cues like windows, words, and shapes on a screen.
+[[ ]] GUIs depend principally on typing text commands.
+[[X]] GUIs are designed with human users in mind.
+[[ ]] GUIs are a second-best alternative for people who aren't smart enough to use the command line.
+[[?]] Hint: There are several correct answers!
+***
+
+<div class = "answer">
+
+Graphical user interfaces don't always offer the same level of control of a system as command line interfaces.  This is why some people strongly advocate that everyone learn to use the command line tools for a system, if possible.
+
+It's true that GUIs depend principally on visual cues like windows, words, and shapes on a screen, and that's why pointing-and-clicking is a frequently used way of interacting with GUIs.
+
+GUIs are designed with **human users** (as opposed to other computer applications) in mind, and there is an entire area of tech devoted to user interface (UI) and user experience (UX) design to make systems and GUIs better.
+
+GUIs do **not** principally rely on typing text commands -- you're thinking of a command line interface!
+
+Finally, and this is important... GUIs are **not** a second best alternative, and using a GUI instead of the command line says nothing about your skills, intelligence, or experience. 
+
+</div>
+***
+
+------
+
+Which of the following is true about the command line interface (CLI)?  Check all that apply!
+
+[[ ]] CLIs are best used by people with great memorization skills.
+[[ ]] CLIs are inherently more dangerous and can do more harm than GUIs, so novice users should avoid using them.
+[[X]] CLIs can feel intimidating to people who are just starting out.
+[[ ]] CLIs depend principally on visual cues like windows, words, and shapes on a screen.
+[[X]] CLIs depend principally on typing text commands.
+[[X]] CLIs can provide greater power and control than GUIs in some systems, like Git.
+[[?]] Hint: There are several correct answers!
+***
+<div class = "answer">
+
+Command line interfaces can indeed feel intimidating to people who are just starting out, in part because some misconceptions about CLIs.  It's not true that CLIs are best used by people with great memorization skills -- many seasoned users of CLIs have to search online for the right command.  It's also not true CLIs are inherently more dangerous and worth avoiding by novice users.  While you can inadvertently do damage with a CLI, you can also inadvertently do damage with any computer interface!  
+
+CLIs depend principally on typing text commands, not on visual cues like windows, words, and shapes on a screen.  And in many cases, CLIs can provide greater power and control than GUIs in some systems.  This is true in the case of Git!
+
+</div>
+***
+
+
+
+## Additional Resources
+
```

Comment:
maybe a resource that gets into more info about command prompt vs git bash vs powershell if you happen to know of a good one if folks want to dive deeper into that since we don't explain their differences? 

It'd be nice to have more than just our module listed here, but also not a huge deal!